backend/server.js

```diff
@@ -328,6 +336,7 @@ sbp('sbp/selectors/register', {
       await removeFromIndexFactory(`_private_resources_${owner}`)(cid)
 
       await sbp('chelonia.db/delete', `_private_hidx=${cid}#0`)
+      await sbp('chelonia.db/delete', `_private_keyop_idx_${cid}_0`)
```

Comment:
Could you add a comment here to explain why this line is here when the code above supposedly deletes these? (i.e. the loop `i > 0` issue might not be obvious to the reader)